Odata-docs/client/v7/navigation-properties.md

```diff
@@ -0,0 +1,91 @@
+
+author: mumbi-o
+ms.author: mowambug
+ms.date: 7/1/2019
+ms.topic: article
+ 
+---
+# Navigation Properties
+**Applies To**: [!INCLUDE[appliesto-odataclient](../../includes/appliesto-odataclient-v7.md)]
+
+When you execute a query, only entities in the addressed entity set are returned. For example, when a query against the Trippin service returns People entities, by default the related Trips entities are not returned, even though there is a relationship between People and Trips. There are two ways to load related entities:
+1. Eager Loading.
+2. Explicit Loading.
+
+## Eager Loading
+You can use the `$expand` query option to request that the query return entities that are related by an association to the entity set that the query requested. 
+
+On OData client you can use the [Expand](/dotnet/api/microsoft.odata.client.dataservicequery-1.expand) method on the [DataServiceQuery&lt;TElement&gt;](/dotnet/api/microsoft.odata.client.dataservicequery-1) to add the `$expand` option to the query that is sent to the data service. You can request multiple related entity sets by separating them by a comma, as in the following example. All entities requested by the query are returned in a single response. The following example returns Trips and Friends together with the People entity set:
+
+``` csharp
+DefaultContainer context = new DefaultContainer(new Uri("https://services.odata.org/V4/(S(uvf1y321yx031rnxmcbqmlxw))/TripPinServiceRW/"));
+DataServiceQuery<Person> query = context.People.Expand("Trips,Friends");
+
+foreach(Person person in query)
+{
+    Console.WriteLine($"First Name: {person.FirstName} Last Name: {person.FirstName}");
+    Console.WriteLine($"***{person.FirstName}'s Trips***");
+    foreach(Trip trip in person.Trips)
+    {
+        Console.WriteLine($"Trip Name: {trip.Name}");
+    }
+    Console.WriteLine($"***{person.FirstName}'s Friends***");
+    foreach (Person friend in person.Friends)
+    {
+        Console.WriteLine($"Friend UserName: {friend.UserName}");
+    }
+    Console.WriteLine("\n\n");
+}
+```
+
+Another way is to use the `AddQueryOption` method like in the example below. 
+
+```csharp
+DefaultContainer context = new DefaultContainer(new Uri("https://services.odata.org/V4/(S(uvf1y321yx031rnxmcbqmlxw))/TripPinServiceRW/"));
+DataServiceQuery<Person> peopleQuery = context.People
+    .AddQueryOption("$expand", "Trips,Friends");
+
+foreach (Person person in peopleQuery)
+{
+    Console.WriteLine($"First Name: {person.FirstName} Last Name: {person.FirstName}");
+    Console.WriteLine($"***{person.FirstName}'s Trips***");
+    foreach (Trip trip in person.Trips)
+    {
+        Console.WriteLine($"Trip Name: {trip.Name}");
+    }
+    Console.WriteLine($"***{person.FirstName}'s Friends***");
+    foreach (Person friend in person.Friends)
+    {
+        Console.WriteLine($"Friend UserName: {friend.UserName}");
+    }
+    Console.WriteLine("\n\n");
+}
+```
+
+## Explicit Loading
```

Comment:
Should this be called lazy loading? 🤔

Reply:
Technically it's `Lazy loading`. But I saw in the WCF docs it was called `Explicit loading`.